vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -39,16 +38,24 @@ public class ExhibitionConfigurationController {
     private ExhibitionFactory exhibitFactory;
 
     @RequestMapping("/staff/exhibit/config")
-    public String showExhibitions(Model model) {
+    public String showExhibitions(HttpServletRequest request, Model model) {
         // for now we assume there is just one exhibition
-        IExhibition exhibition = exhibitManager.getStartExhibition();
-        if(exhibition!=null) {
-            model.addAttribute("exhibition", exhibition);
+        Exhibition exhibition = (Exhibition) exhibitManager.getStartExhibition();
+        String previewId = null;
+        ExhibitionModes exhibitionMode = null;
+        if (exhibition == null) {
+            Exhibition exhibitionObj = (Exhibition) exhibitFactory.createExhibition();
+            model.addAttribute("exhibition", exhibitionObj);
+            previewId = exhibitionObj.getPreviewId();
+            exhibitionMode = exhibitionObj.getMode();
         } else {
-            model.addAttribute("exhibition", new Exhibition());
+            previewId = exhibition.getPreviewId();
+            exhibitionMode = exhibition.getMode();
         }
         model.addAttribute("exhibitionModes", Arrays.asList(ExhibitionModes.values()));
         model.addAttribute("spacesList", spaceRepo.findAll());
+        model.addAttribute("previewId", previewId);
+        model.addAttribute("exhibitionMode",exhibitionMode.name());
```

Comment:
once the exhibition object is in the model, these two lines are unnecessary, aren't they?